docs/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ### next
 
+  * tpm2_load:
+      - Added capability to read public and private portions from PEM files
```

Comment:
This might sound apt - "Added capability to load pem files in TSS2-Private-Key format for interoperability with tpm2-tss-engine and tpm2-openssl provider"